ceph.spec.in

```diff
@@ -488,8 +488,18 @@ BuildArch:      noarch
 Group:          System/Filesystems
 %endif
 Requires:       ceph-mgr = %{_epoch_prefix}%{version}-%{release}
+%if 0{?fedora} || 0%{?suse_version}
```

Comment:
Looks like this might be a syntax error here, "%if 0{?fedora}"

We need this to be `%if 0{?fedora}"

And we also need RHEL 8, so `%if 0{?fedora} || 0%{?rhel} > 7 || 0%{?suse_version}`